model/device_deployment.go

```diff
@@ -194,6 +194,10 @@ func (stat DeviceDeploymentStatus) Active() bool {
 		stat <= DeviceDeploymentStatusActiveHigh
 }
 
+func (stat DeviceDeploymentStatus) Successful() bool {
+	return stat == DeviceDeploymentStatusSuccess
```

Comment:
what about NoArtifact and AlreadyInstalled?

Reply:
I removed the method and we just store all the statuses.